### ISSUE TITLE
Remove deprecated /apps/*/channels/*/events API call

### ISSUE THREAD

**Issue by jameshfisher** [lukeredpath/libPusher#205]
I noticed this library is using a deprecated resource in the Pusher API: https://pusher.com/docs/rest_api_deprecated#method-post-event

Here's where it uses that resource: https://github.com/lukeredpath/libPusher/blob/f64d469fa80d8203598924ee35bad88b83261959/Library/PTPusherAPI.m#L37

It should be updated to use this resource: https://pusher.com/docs/rest_api#method-post-event

**Comment by lukeredpath** [comment]
Thanks. I'm still considering whether to keep PTPusherAPI in the library to be honest. It has dependencies on some stuff I'd like to get rid of and is only a wrapper around a single API call I'm wondering if it has any value. Especially given how easy it is to trigger events over the socket-based API which is what this library should really be focussed on IMO.

@hamchapman what do you think?

**Comment by hamchapman** [comment]
Yeah I'd be tempted to remove it. Keeping libPusher focussed on the socket-based API is a good move in my books
